globals/game.gd

```diff
@@ -28,18 +28,17 @@ func set_mode(p_mode):
 func mouse_enter(obj):
 	var text
 	var tt = obj.get_tooltip()
-	if current_tool != null:
-		text = "use.combine_id"
-		if current_action != "":
-			text = tr(current_action + ".combine_id")
-		text = text.replace("%2", tr(tt))
-		text = text.replace("%1", tr(current_tool.get_tooltip()))
-	elif obj.inventory:
+
+	#TODO: Fix inventory combination
+	#if current_action == "pick_up" && text != null:
+		#text = tr(current_action + ".combine_id")
+		#text = text.replace("%2", tr(tt))
+		#text = text.replace("%1", tr(current_tool.get_tooltip()))
+	if obj.inventory:
```

Comment:
Looks like this file was missing the changes from 9139c386bff4c89136fdda0b2bbc4524ef36bc31 and it breaks interaction in some places.